### DISCUSSION TITLE
08-05-2024 Steering Committee Meeting Minutes

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#138] [Steering Committee]
1. Progress and Review of Setting Up DCDAO on Tally:
  - Discuss the progress of setting up DCDAO on Tally.
  - Reference: [GitHub Issue #4](https://github.com/dcdao/contracts/issues/4).

2. Review Reports and Discussions on the Forum:
  - Review recent reports and discussions from the DCDAO forum.

3. Request Round Table Time from Aki:
  - Review and request a round table session with Aki.
  - Reference: [GitHub Discussion #131](https://github.com/orgs/dcdao/discussions/131).

4. Review InvArch DAO Page:
  - Review the DAO page created on InvArch and understand how it works.
  - Request more information and updates if available.
  - Reference: [InvArch Network Twitter Post](https://twitter.com/InvArchNetwork/status/1785309072155115876).

**Comment by 2xJDubs** [comment]
@hackfisher what kind of information are you looking for on the InvArch/Tinkernet DAO staking?  Saso or I can help explain thing there.
